app/views/users/_pending.html.erb

```diff
@@ -0,0 +1,13 @@
+<li>
+  <div class="d-flex">
+    <h2>Name:</h2>
+    <%= user.name %>
+  </div>
+  <div class="d-flex">
+    <h2>Name:</h2>
+    <%= user.name %>
+  </div>
+  <span>
+    <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
```

Comment:
- [ ] A user can receive more than one friendship request, and from these requests, I can accept one and decline the other. This should not be the case, let's fix this bug.

![requests](https://user-images.githubusercontent.com/53124548/133873680-0a295b15-9ae6-4752-b8dd-0f6f3c9775c9.png)